projects/fsastorefrontlib/src/cms-components/questionnaire-carousel/questionnaire-carousel.component.html

```diff
@@ -84,3 +87,15 @@ <h4 class="mb-4">{{ product.name }}</h4>
 <ng-template #finalResult>
```

Comment:
You can remove this template since the faces always exists now.

Reply:
removed